### PR TITLE
Flash messages

### PR DESCRIPTION
**Pull request by apanjain**

### CODE REVIEW

File: frontend/src/routes/Queries/contact.js [hunk 1]
```diff
@@ -33,6 +34,7 @@ class Contact extends Component {
     }
     handleSubmit = (e) => {
         e.preventDefault(); 
+        this.setState({isLoading:true})
```
Comment (hacksdump) [inline frontend/src/routes/Queries/contact.js:37]: Don't send `isLoading` so early. Set only when an actual request is sent.
Currently, it is showing even when captcha verification is not complete. In such a case, only show the error popup. The loading screen won't go away and form can't be filled again.

File: frontend/src/components/IsLoading/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+            <>
+            <div className={styles.mainDiv}>
+                    <img className={styles.image} src={Loader} alt="Loading..."/>
+            </div>
+            </>
+         );
+    }
+}
+ 
+export default IsLoading;
```
Comment (hacksdump) [inline frontend/src/components/IsLoading/index.js:16]: Rename `IsLoading` to `LoadingOverlay`

File: frontend/src/components/Flash/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    useEffect(() => {
+        if(document.querySelector('.close') !== null) {
+            document.querySelector('.close').addEventListener('click', () => setVisibility(false));
+        }
+    })
+
+    return (
+        visibility &&
+            <div className={styles.alert +' ' +styles[type]}>
+                    <div className={styles.cross + ' close'}>
```
Comment (hacksdump) [inline frontend/src/components/Flash/index.js:32]: This cross button can be factored out into a component of its own in the Flash directory.

File: frontend/src/routes/Queries/contact.js [hunk 4]
```diff
@@ -44,19 +46,24 @@ class Contact extends Component {
             .then((response) => {
```
Comment (hacksdump) [inline frontend/src/routes/Queries/contact.js:46]: Move API call to `src/api/methods/index.js`

**Comment by apanjain** [comment]
Changes have been made

**Pull request merged**